sync/rest/maps/update-map-item/update-map-item.7.x.java

```diff
@@ -17,12 +16,14 @@
   public static void main(String[] args) {
     Twilio.init(ACCOUNT_SID, AUTH_TOKEN);
 
-    Map<String, Object> data = new HashMap<>();
+    HashMap<String, Object> data = new HashMap<String, Object>();
```

Comment:
It's good practice to leave the parent type on the left.  the generic types are inferred on the right side, so they are not needed. You can write:
`Map<String, Object> data = new HashMap<>();`